main.py

```
@@ -1210,6 +1224,15 @@ async def lounge_host(update: Update, context: ContextTypes.DEFAULT_TYPE):
                     except: pass
                     return
 
+                elif not ANTIGRAVITY_BYPASS_PASSWORD:
```

Ruff (0.15.12)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)